### PR TITLE
Vue Clean Up Docs

### PR DESCRIPTION
**Pull request by ErikCH**
*Issue #, if available:* Cleanup Vue Docs

*Description of changes:*
Clean up Vue Docs

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: examples/vue/src/pages/ui/components/authenticator/sign-in-federated/index.vue [hunk 1]
```diff
@@ -1,22 +1,17 @@
...
 </script>
+
+<template>
+  <authenticator :login-mechanisms="['email', 'facebook', 'amazon', 'google']">
+    <template v-slot="{ user, signOut }">
+      <h1>Hello {{ user.username }}!</h1>
+      <button @click="signOut">Sign Out</button>
+    </template>
+  </authenticator>
+</template>
```
Comment (ErikCH) [inline examples/vue/src/pages/ui/components/authenticator/sign-in-federated/index.vue:17]: Just some minor cleanup of the examples.

File: docs/src/pages/ui/getting-started/installation/installation.vue.mdx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import BlockSwitcher from 'amplify-docs/src/components/BlockSwitcher';
 
 [![version](https://img.shields.io/npm/v/@aws-amplify/ui-vue/next.svg)](https://www.npmjs.com/package/@aws-amplify/ui-angular/v/next)
 
-Install `@aws-amplify/ui-vue@next` with [npm](https://www.npmjs.com/) or [yarn](https://yarnpkg.com/):
+For Vue 3 install `@aws-amplify/ui-vue@next` with [npm](https://www.npmjs.com/) or [yarn](https://yarnpkg.com/):
```
Comment (ErikCH) [inline docs/src/pages/ui/getting-started/installation/installation.vue.mdx:6]: I wanted to explicitly call out this package is only for Vue 3

File: docs/src/pages/ui/components/authenticator/sign-up-with-phone.react.mdx [hunk 3]
```diff
@@ -0,0 +1,3 @@
+```js{3} file=../../../../../../examples/next/pages/ui/components/authenticator/sign-up-with-phone/index.page.tsx#L7-L19
+
+```
```
Comment (ErikCH) [inline docs/src/pages/ui/components/authenticator/sign-up-with-phone.react.mdx:3]: The line numbers weren't right, so I changed them

File: docs/src/pages/ui/components/authenticator/index.page.mdx [hunk 4]
```diff
@@ -41,25 +41,21 @@ based on a unique `username` using [`Auth.signUp`](https://docs.amplify.aws/lib/
...
 
-The `Authenticator` automatically infers [`email` as a login mechanism](https://docs.amplify.aws/console/auth/authentication#to-configure-how-users-log-in-to-an-app).
-
 You can explicitly configure Amplify to match your backend via:
 
-```js{4-10} file=../../../../../../examples/next/pages/ui/components/authenticator/sign-up-with-email/index.page.tsx#L2-L10
-
-```
+<Fragment>
+  {({ platform }) => import(`./sign-up-with-email.${platform}.mdx`)}
```
Comment (ErikCH) [inline docs/src/pages/ui/components/authenticator/index.page.mdx:47]: Now we can support specific fragments for each platform

Comment (ErikCH) [reply docs/src/pages/ui/components/authenticator/index.page.mdx:47]: For the examples

**Review by eddiekeller — APPROVED** [review]

File: docs/src/pages/ui/getting-started/installation/installation.vue.mdx (hunk 2 above)
Comment (wlee221) [reply docs/src/pages/ui/getting-started/installation/installation.vue.mdx:6]: Makes sense!

**Review by wlee221 — APPROVED** [review]
🚢

**Pull request merged**